k2/csrc/fsa_algo.cu

```diff
@@ -141,6 +141,19 @@ bool Intersect(FsaOrVec &a_fsas, FsaOrVec &b_fsas,
     }
     // the check on the previous line will fail.
   }
+  if (properties_a < 0) {
+    properties_a = GetFsaBasicProperties(a_fsas);
+  }
+  if (properties_b < 0) {
+    properties_b = GetFsaBasicProperties(b_fsas);
```

Comment:
Noted we'll get mutiple testing failure here as we are calling `GetFsaBasicProperties` for a FsaVec, it always return 0. I have fixed it locally, will push together with my other changes later

Reply:
I mean, we should call `GetFsaVecBasicProperties` for FsaVec.

Reply:
Fixed in https://github.com/k2-fsa/k2/pull/446.